chpl/chpl-service/src/main/java/gov/healthit/chpl/activity/CorrectActionPlanActivityMetadataBuilder.java

```diff
@@ -0,0 +1,20 @@
+package gov.healthit.chpl.activity;
+
+import org.springframework.stereotype.Component;
+
+import gov.healthit.chpl.domain.activity.ActivityMetadata;
+import gov.healthit.chpl.dto.ActivityDTO;
+
+@Component("correctActionPlanActivityMetadataBuilder")
+public class CorrectActionPlanActivityMetadataBuilder extends ActivityMetadataBuilder {
+
+    public CorrectActionPlanActivityMetadataBuilder() {
+        super();
+    }
+
+    @Override
+    protected void addConceptSpecificMetadata(ActivityDTO dto, ActivityMetadata metadata) {
+
```

Comment:
I guess this is fine if we don't have anything to add here... does the UI even get corrective action activity reports? Maybe we don't need this?

Reply:
There are existing corrective action report activity logged in the user actions report, but there is no correct actions activity report.